src/util.ts

```diff
@@ -917,28 +917,41 @@ export function runGodoc(packagePath: string, symbol: string, token: vscode.Canc
 		return Promise.reject(new Error('Cannot find "go" binary. Update PATH or GOROOT appropriately'));
 	}
 
-	const cwd = packagePath;
-	const getCurrentPackagePromise = 1 !== 1 ? getCurrentPackage(cwd) : Promise.resolve(cwd);
+	const getCurrentPackagePromise = path.isAbsolute(packagePath) ? getCurrentPackage(packagePath) : Promise.resolve(packagePath);
 	return getCurrentPackagePromise.then(packageImportPath => {
 		return new Promise<string>((resolve, reject) => {
 			const env = getToolsEnvVars();
 			const args = ['doc', '-c', '-cmd', '-u', packageImportPath, symbol];
-			const p = cp.execFile(goRuntimePath, args, { env }, (err, stdout) => {
+			const p = cp.execFile(goRuntimePath, args, { env }, (err, stdout, stderr) => {
 				if (err) {
-					return reject(err);
+					return reject(err.message || stderr);
 				}
 				const godocLines = stdout.split('\n');
-				let doc = '';
 
-				for (let i = 1; i < godocLines.length && godocLines[i].startsWith('    '); i++) {
-					doc += godocLines[i].substring(4) + '\n';
+				// Skip trailing empty lines
+				let lastLine = godocLines.length - 1;
+				for (; lastLine > 1; lastLine--) {
+					if (godocLines[lastLine].trim()) {
+						break;
+					}
+				}
+
+				let doc = '';
+				for (let i = 1; i <= lastLine; i++) {
```

Comment:
With `<=`, aren't you also entering this loop with the first trailing empty line? Is this intended?

Reply:
`lastLine` points to last non empty line not the first empty line. So we should be good.

Reply:
Sorry I misread the code. 😓 
You are right.